build/make-distribution.sh

```diff
@@ -127,7 +127,7 @@ function build_service {
   # Store the command as an array because $MVN variable might have spaces in it.
   # Normal quoting tricks don't work.
   # See: http://mywiki.wooledge.org/BashFAQ/050
-  BUILD_COMMAND=("$MVN" clean package -DskipTests $@)
```

Comment:
Clean is necessary. It will clean all remaining build files and create a reproducible result.

Reply:
+1, please reserve clean, dirty cache may cause fatal issues